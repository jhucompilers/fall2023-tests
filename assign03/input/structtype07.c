struct Foo {
  int x;
};

int main(void) {
  struct Foo f1;
  struct Foo f2;

  struct Foo *arr[2];
  arr[0] = &f1;
  arr[1] = &f2;

  arr[0]->x = 99;
  arr[1]->x = 101;

  return 0;
}
