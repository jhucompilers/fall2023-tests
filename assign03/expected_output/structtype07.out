0|struct Foo|type|struct Foo {}
1|x|variable|int
0|main|function|function () returning int
2|f1|variable|struct Foo {int}
2|f2|variable|struct Foo {int}
2|arr|variable|array of 2 x pointer to struct Foo {int}
